determiner.py

```diff
@@ -0,0 +1,115 @@
+#! /usr/bin/env python3
+
+import argparse
+import binascii
+import os
+import sys
+
+## print header first 
+authorSignature = '\ndeterminer.py - Published by Laura Tzigiannis and Moritz Nentwig\n'
+authorSignature += '---------------------------------------------------------------\n'
+print(authorSignature)
+
+## tests
+def check_file_extension(file): 
+    _, file_extension = os.path.splitext(file)
+    if file_extension == ".aes":
+        return("AES")
+    elif file_extension == ".des":
+        return("DES")
+    elif file_extension == ".3des":
+        return("3DES")
+    elif file_extension == ".idea":
+        return("IDEA")
+    elif file_extension == ".bf":
+        return("Blowfish")
+    else:
+        return("Unrecognized file extension")
+
+def check_file_header(file):
+    encryption_algorithms = ["AES", "DES", "3DES", "IDEA", "Blowfish"]
```

Comment:
It's not working for Blowfish. I think you have to weite it in capital letters

Reply:
And it doesn't work for 3des, since it finds des first and returns in loop. should be resolved by just switching 3des & des in the array